### PR TITLE
Jamulus -h correction

### PR DESCRIPTION
**Pull request by gilgongo**
Put -p and -Q in correct place.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -881,6 +878,9 @@ QString UsageArguments ( char **argv )
         "  -m, --htmlstatus      enable HTML status file, set file name\n"
         "  -o, --serverinfo      infos of this server in the format:\n"
         "                        [name];[city];[country as QLocale ID]\n"
+        "  -p, --port            set the local port number\n"
+        "  -Q, --qos             request quality of service in the IP header octet\n"
```
Comment (hoffie) [inline src/main.cpp:882]: Shouldn't it be p (port) < P (delaypan) < Q (qos) (i.e. -Q after -P)?

Comment (gilgongo) [reply src/main.cpp:882]: Oops

**Review by ann0see — APPROVED** [review]
Looks good to me. Squash and merge?

**Comment by softins** [comment]
See https://github.com/jamulussoftware/jamulus/commit/377e6ec7e259332b856358520c5c4aff535d59a4#commitcomment-49357651

**Pull request merged**